.github/workflows/android-monkey.yaml

```diff
@@ -76,25 +76,17 @@ jobs:
         run: |
           cmake --version
           ninja --version
-          ./gradlew -Pfirebase -Parm64-v8a -Parmeabi-v7a -Px86_64 assembleGoogleDebug uploadCrashlyticsSymbolFileGoogleDebug
+          ./gradlew -Pfirebase -Parm64 -Parmeabi-v7a assembleGoogleBeta uploadCrashlyticsSymbolFileGoogleBeta uploadCrashlyticsMappingFileGoogleBeta
 
       - name: Run monkey
         run: |
           gcloud auth activate-service-account --key-file android/app/firebase-test-lab.json
           gcloud config set project omapsapp
-          gcloud firebase test android run --app ./android/app/build/outputs/apk/google/debug/OrganicMaps-*-google-debug.apk \
+          gcloud firebase test android run --app ./android/app/build/outputs/apk/google/beta/OrganicMaps-*-google-beta.apk \
             --device model=husky,version=34 \
             --device model=cheetah,version=33 \
             --device model=bluejay,version=32 \
             --device model=a51,version=31 \
             --device model=f2q,version=30 \
             --device model=a10,version=29 \
```

Comment:
Previously, Android 5, 6, 7 devices were tested. Can we leave them too to discover regression crashes?